llms.txt

```diff
@@ -9122,6 +9122,9 @@ Doc-Content: https://docs.kluster.ai/tutorials/klusterai-api/effective-prompt-en
     "api_key = getpass('Enter your kluster.ai API key: ')\n",
     "\n",
     "# Feel free to swap out the model with any of Kluster's supported models\n",
+    "# You can also use your own Kluster dedicated deployment after launching one\n",
```

Comment:
Just seeing this on the GitHub view, not sure what the context here is but there's capitalized kluster in a few places - should be kluster.ai everywhere